packages/uif/src/reducer/handlers/handleRetryTick.ts

```diff
@@ -0,0 +1,17 @@
+import assert from 'node:assert'
+
+import { RetryTickAction } from '../types/IndexerAction'
+import { IndexerReducerResult } from '../types/IndexerReducerResult'
+import { IndexerState } from '../types/IndexerState'
+
+export function handleRetryTick(
+  state: IndexerState,
+  _action: RetryTickAction,
+): IndexerReducerResult {
+  assert(state.retryingTick, 'should be retrying tick')
```

Comment:
Maybe assert idle state? Also what happens if someone Ticks before we retry?

Reply:
1. good idea
1. In `handleRequestTick` we have a logic blocking tick (using `retryingTick`)